source/docs/software/telemetry/robot-telemetry-with-annotations.rst

```diff
@@ -4,7 +4,7 @@ New for 2025: Robot Telemetry with Annotations (Java Only)
 At its most basic, telemetry with annotations can be accomplished with only a few lines of code:
 
 #. Add the ``@Logged`` annotation to your main robot class and to the classes it references, like subsystems
-#. Bind the logger to the robot in ``robotInit()`` to start logging
+#. Bind the logger to the robot in ``Robot()`` to start logging
```

Comment:
This should reference the constructor, rather then Robot, since it's possible to have different names.